Examples/Scripts/MaterialMaping/Mat_map_surface_plot_dist.C

```diff
@@ -39,7 +39,7 @@ void plot(TGraph* Dist, const sinfo& surface_info, const std::string& name){
   surface_r->SetNDC();
 
   // Disk
-  if(surface_info.type == 2){
+  if(surface_info.type == 2 || surface_info.type == 4){
```

Comment:
Why does 4 appear here now?

Reply:
I forgot I had those changes there and included all the changes in the Script, I can put that in a different PR if need (since they are just script changes I don't think that is needed but let me know). When I was trying to do some mapping of the material directly onto the sensor I noticed that the script didn't work if we map onto trapezoid surface so I changed added the option.